server/logs/log_exports.lua

```diff
@@ -0,0 +1,15 @@
+exports("LogMessage", function(level, message, metadata)
+	Logs:LogMessage(level, message, metadata)
+end)
+
+exports("LogInfo", function(message, metadata)
+	Logs:LogInfo(message, metadata)
```

Comment:
I'm still not sure about this tbh, I dont want people getting used to a ton of different exports. 

Everything we are doing should serialize well between languages afaik, just passing the logger instance I think would be better since they would only have to call the export once instead of on every log.

Or just have one "LogMessage" export they can use instead of the multiple like below.

This will be an issue for analytics as well, tons of different exports vs just passing some Analytics instance they can call methods on. 

What are your thoughts?

Reply:
Yeah, I think we can have export that is LogMessage, and then the instance. That sounds good.